include/ofi_mem.h

```diff
@@ -410,16 +410,13 @@ static inline void util_buf_indexed_release(struct util_buf_pool *pool, void *bu
 static inline size_t util_get_buf_index(struct util_buf_pool *pool, void *buf)
 {
 	assert(util_buf_get_ftr(pool, buf)->region->num_used);
-	assert(pool->attr.indexing);
```

Comment:
Should `attr.indexing` be renamed to something else since getting the buf index is not dependent on it?

Reply:
I don't what the name can be applicable here.

What do you think about something like `return_lowest_index` or `maintain_sorted_indices`?

Reply:
@a-ilango Do you have any proposals regarding my comment above?

Reply:
So `indexing` is only used for maintaining a sorted list to help AV? We can rename it to `ordered` or `maintain_order` and add a comment in the definition as well.

Reply:
Thanks, @a-ilango. Please, find the changes in the e1b3e3d4d09dda5505ddecd48074dfc0dba778f9. Also minor changes were in RxD -- @aingerson, please take a look.